get_team_roles/team_roles.py

```diff
@@ -0,0 +1,30 @@
+import argparse
+import sys
+import pdpyras
+
+def get_teams(session, comma_separated):
+    if comma_separated:
+        sys.stdout.write("Team ID, Team Name, User ID, User name, Team role\n")
+    for team in session.iter_all('teams'):
+        get_team_members(team['id'], team['name'], session, comma_separated)
```

Comment:
We should add some error handling to both API calls, especially because this is public. You can do this with pdpyras: https://pagerduty.github.io/pdpyras/#error-handling

I think once there's some good error handling, we'll be fine to forgo retry logic since the script itself isn't doing too much.